sql/pg_duckdb--0.0.1.sql

```diff
@@ -1,38 +1,38 @@
 LOAD 'pg_duckdb';
 
-CREATE OR REPLACE FUNCTION read_parquet(path text)
+CREATE OR REPLACE FUNCTION public.read_parquet(path text)
```

Comment:
I'm adding public here so that these functions get created in a consistent schema.

Reply:
undid this change now that I find the functions in a different way.